hack/update-pkg-manifest.sh

```diff
@@ -13,82 +13,87 @@
-  echo "Install bazel at https://bazel.build" >&2
+if [[ -z "${BUILD_WORKSPACE_DIRECTORY:-}" ]]; then
+  echo 'Must be run via "make release/update-pkg-manifest"' >&2
   exit 1
-else
-  (
-    set -o xtrace
-    bazel run //hack:update-pkg-manifest
-  )
-  exit 0
 fi
 
-opsdk=$(realpath "$1")
-kstomize="$(realpath "$2")"
-opm=$(realpath "$3")
-faq="$(realpath "$4")"
-export PATH=$(dirname "$opsdk"):$PATH
-# This script should be run via `bazel run //hack:update-pkg-manifest
-# It will be used on Openshift certification image bundle releases.
-# It created a bundle in the package manifests format.
-# We are keeping the format that was initially used.
-REPO_ROOT=${BUILD_WORKSPACE_DIRECTORY}
-cd "${REPO_ROOT}"
-echo ${REPO_ROOT}
-echo "+++ Running update package manifest for certification"
-[[ -z "$5" ]] && { echo "Error: RH_BUNDLE_VERSION not set"; exit 1; }
-RH_BUNDLE_VERSION="$5"
-echo "RH_BUNDLE_VERSION=$RH_BUNDLE_VERSION"
-[[ -z "$6" ]] && { echo "Error: RH_COCKROACH_OP_IMG not set"; exit 1; }
-RH_COCKROACH_OP_IMG="$6"
-echo "RH_COCKROACH_OP_IMG=$RH_COCKROACH_OP_IMG"
-[[ -z "$7" ]] && { echo "Error: RH_PKG_MAN_OPTS not set"; exit 1; }
-RH_PKG_MAN_OPTS="$7"
-echo "RH_PKG_MAN_OPTS=$RH_PKG_MAN_OPTS"
-[[ -z "$8" ]] && { echo "Error: RH_COCKROACH_DATABASE_IMAGE not set"; exit 1; }
-RH_COCKROACH_DATABASE_IMAGE="$8"
-echo "RH_COCKROACH_DATABASE_IMAGE=$RH_COCKROACH_DATABASE_IMAGE"
-DEPLOY_PATH="deploy/certified-metadata-bundle/cockroach-operator"
-DEPLOY_CERTIFICATION_PATH="deploy/certified-metadata-bundle"
-if [ -d "${DEPLOY_PATH}/${RH_BUNDLE_VERSION}" ]
-then
-    echo "Folder ${DEPLOY_PATH}/${RH_BUNDLE_VERSION} already exists. Please increase the version or remove the folder manually."
+export PATH="$(pwd)/hack/bin:${PATH}"
+
+main() {
+  local rh_bundle_version="${1}"
+  local rh_operator_image="${2}"
+  local rh_package_options="${3}"
+  local rh_crdb_image="${4}"
+  local cert_path="deploy/certified-metadata-bundle"
+  local deploy_path="${cert_path}/cockroach-operator"
+
+  echo "+++ Running update package manifest for certification"
+  echo "RH_BUNDLE_VERSION=${rh_bundle_version}"
+  echo "RH_OPERATOR_IMAGE=${rh_operator_image}"
+  echo "RH_PACKAGE_OPTS=${rh_package_options}"
+  echo "RH_CRDB_IMAGE=${rh_crdb_image}"
+  echo "CERTIFICATION_PATH=${cert_path}"
+  echo "DEPLOY_PATH=${deploy_path}"
+
+  cd "${BUILD_WORKSPACE_DIRECTORY}"
+  ensure_unique_deployment "${deploy_path}/${rh_bundle_version}"
+  generate_package_manifest "${rh_bundle_version}" "${rh_package_options}" "${deploy_path}"
+  generate_csv "${deploy_path}/${rh_bundle_version}" "${rh_operator_image}"
+  combine_files "${deploy_path}/${rh_bundle_version}" "${rh_bundle_version}"
+}
+
+ensure_unique_deployment() {
+  if [ -d "${1}" ]; then
+    echo "Folder ${1} already exists. Please increase the version or remove the folder manually." >&2
     exit 1
-fi
-rm -rf ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}
-"$opsdk" generate kustomize manifests -q --verbose
```

Comment:
Need to bring this back. The trouble is that it's already not working on master.

Reply:
I was able to resolve this. The issue was two-fold:

1. We had the wrong `group` value in PROJECT
2. Because we use `/apis`, rather than `/api` for this single group project, we needed to add `--apis-dir` to the call.

Reply:
We use "apis" rather than "api" because our operator framework was not generating properly.